stubs/Caddyfile

```diff
@@ -5,6 +5,5 @@ fastcgi / 127.0.0.1:9000 php {
 }
 
 rewrite {
-    r .*
-    to /server.php?{query}
+    to {path} {path}/ /server.php?{query}
```

Comment:
The `to` change may not be quite what is needed in Valet's case, but I do believe the `r .*` is redundant, since by default, `rewrite` matches all requests.